app/mailers/visitor_mailer.rb

```diff
@@ -21,27 +21,49 @@ def booking_confirmation_email(visit, confirmation, token)
     @confirmation = confirmation
     @token = token
 
-    mail(from: sender, reply_to: prison_mailbox_email, to: recipient, subject: "Visit confirmed: your visit for #{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} has been confirmed")
+    mail(
+      from: sender,
+      reply_to: prison_mailbox_email,
+      to: recipient,
+      subject: "Visit confirmed: your visit for " \
+       "#{Date.parse(@slot.date).strftime('%e %B %Y').gsub(/^ /,'')} "\
+       "has been confirmed"
```

Comment:
<a href="http://guides.rubyonrails.org/i18n.html#translations-for-action-mailer-e-mail-subjects">default_i18n_subject</a> :zap: